cpp/CMakeLists.txt

```diff
@@ -52,8 +52,8 @@ if((CMAKE_CXX_COMPILER_ID STREQUAL "GNU") AND(CMAKE_CXX_COMPILER_VERSION VERSION
     endif(MEMILIO_SANITIZE_UNDEFINED)
 
     if(MEMILIO_SANITIZE_ADDRESS OR MEMILIO_SANITIZE_UNDEFINED)
-        string(APPEND CMAKE_CXX_FLAGS_DEBUG " -fno-omit-frame-pointer -fno-sanitize-recover=all")
-        string(APPEND CMAKE_LINKER_FLAGS_DEBUG " -fno-omit-frame-pointer -fno-sanitize-recover=all")
+        string(APPEND CMAKE_CXX_FLAGS_DEBUG " -fno-omit-frame-pointer")
+        string(APPEND CMAKE_LINKER_FLAGS_DEBUG " -fno-omit-frame-pointer")

```

Comment:
This can be omitted because we implement sanitizers.cpp, am i right?

Reply:
no, this is something different.